src/dbnode/persist/fs/msgpack/decoder.go

```diff
@@ -41,13 +43,17 @@ var (
 	emptyLogEntry               schema.LogEntry
 	emptyLogMetadata            schema.LogMetadata
 	emptyLogEntryRemainingToken DecodeLogEntryRemainingToken
-)
 
-var errorUnableToDetermineNumFieldsToSkip = errors.New("unable to determine num fields to skip")
+	errorUnableToDetermineNumFieldsToSkip          = errors.New("unable to determine num fields to skip")
+	errorCalledDecodeBytesWithoutByteStreamDecoder = errors.New("called decodeBytes with out byte stream decoder")
```

Comment:
nit: rename this to `errThing... =` rather than `errorThing... = ` for consistency with other packages/etc.